api/src/users/users.controller.ts

```diff
@@ -204,6 +204,44 @@ export class UsersController {
     }
   }
 
+  @Post('sent-resend-link')
```

Comment:
Change the route name to send instead of sent

Reply:
route name changed